registration/migrations/0005_auto_20210310_0120.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.1.6 on 2021-03-10 01:20
```

Comment:
can we squash these migration files so that the `related_name` addition is just done in one file? 😄

Reply:
Squashed the 3 migration files to: 0005_auto_20210310_0120_squashed_0007_auto_20210310_0152

I kept the default name, not sure if we want to rename it or not?